jwst/ramp_fitting/ramp_fit.py

```diff
@@ -2895,7 +2905,9 @@ def fit_2_group(slope_s, intercept_s, variance_s, sig_intercept_s,
     if (len(wh_sat1[0]) > 0):
         data0_slice = data[0, :, :].reshape(npix)
         slope_s[wh_sat1] = data0_slice[wh_sat1]
-        variance_s[wh_sat1] = 0.
+        # set variance non-zero because calling function uses variance=0 to
```

Comment:
Note to reviewers: this is the only actual change in the code needed to fix the bug. All the rest is cosmetic.